test-two/src/services/github-test.js

```diff
@@ -1,5 +1,12 @@
 import { getUsersForOrganisation } from "./github";
 import * as http from "../helpers/http";
+import { joinPaginatedRequests } from "../helpers/http";
```

Comment:
not sure if this is being used either, looks like everything is referencing joinPaginatedRequests via http?

Reply:
same as the one before, I caught that with ts and eslint...